packages/components/radio-group/src/radio.scss

```diff
@@ -125,3 +73,49 @@ circle {
     }
   }
 }
+
```

Comment:
do you put the loops at the top because they're only setting variables?

Reply:
Yes, variables at the top, then the styles that use the variables.

Reply:
Otherwise you have some variables at the top and some at the bottom, and it's kinda not clear where you have to look.